### PR TITLE
[full-ci] Disallow to share the personal drive via graph

### PR DESCRIPTION
**Pull request by 2403905**
## Description

Disallow to share the personal drive via graph API

## Related Issue

- Fixes [https://github.com/owncloud/ocis/issues/8494](https://github.com/owncloud/ocis/issues/8494)
- Related fix [https://github.com/cs3org/reva/pull/4559](https://github.com/cs3org/reva/pull/4559)
## How Has This Been Tested?

- test environment:
- test case 1:
- test case 2:
- ...

## Types of changes

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Technical debt
- [ ] Tests only (no source changes)

## Checklist:

- [X] Code changes
- [ ] Unit tests added
- [ ] Acceptance tests added
- [ ] Documentation ticket raised: <link>

### CODE REVIEW

File: services/graph/pkg/service/v0/driveitems.go [hunk 1]
```diff
@@ -469,6 +469,10 @@ func (g Graph) Invite(w http.ResponseWriter, r *http.Request) {
 
 	condition := unifiedrole.UnifiedRoleConditionGrantee
 	if IsSpaceRoot(statResponse.GetInfo().GetId()) {
+		if statResponse.GetInfo().GetSpace().GetSpaceType() == _spaceTypePersonal {
```
Comment (rhafer) [inline services/graph/pkg/service/v0/driveitems.go:472]: I think this check should live in the shareprovider (https://github.com/cs3org/reva/blob/edge/internal/grpc/services/usershareprovider/usershareprovider.go#L159)

**Review by rhafer — CHANGES_REQUESTED** [review]

**Comment by kulmann** [comment]
Same question here as in the other PR: target bugfix at `stable-5.0`? cc @micbar

**Comment by micbar** [comment]
target `master`& backport

File: services/graph/pkg/service/v0/driveitems.go (hunk 1 above)
Comment (2403905) [reply services/graph/pkg/service/v0/driveitems.go]: @rhafer Moved to reva. Please review [https://github.com/cs3org/reva/pull/4559](https://github.com/cs3org/reva/pull/4559)

**Review by rhafer — APPROVED** [review]

**Pull request merged**